coordinate.cpp

```diff
@@ -49,35 +61,41 @@ void test_coordinate() {
 
     double default_speed = 1;
     c.move(direction);
-    assert(get_x(c) == default_speed);
-    assert(get_y(c) == 0 );
+    double error_x = abs(get_x(c) - default_speed);
+    double error_y = abs(get_y(c) - 0);
+    assert(error_x < 0.00001);
+    assert(error_y < 0.00001);
```

Comment:
We don't really need this bit. Line 69 covers all this.

Reply:
removed